sdk/search/Azure.Search.Documents/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Release History
 
-## 11.3.0-beta.1 (Unreleased)
+## 11.3.0-beta.1 (2021-04-06)
 
+### Added
+- Added support for [`Azure.Core.GeoJson`](https://docs.microsoft.com/dotnet/api/azure.core.geojson) types in `SearchDocument`, `SearchFilter` and `FieldBuilder`.
+- Added [`EventSource`](https://docs.microsoft.com/dotnet/api/system.diagnostics.tracing.eventsource) based logging. Event source name is **Azure-Search-Documents**.
```

Comment:
Should we say it's specifically for tuning SearchIndexingBufferedSender batch sizes?  Can we point people at a quick sample showing how to enable it?

Reply:
I don't see a good sample. I will add one in our repo.